### PR TITLE
DBZ-4832 Don't set truststore/keystore parameters to system variables

### PR DESCRIPTION
**Pull request by harveyyue**
https://issues.redhat.com/browse/DBZ-4832

### CODE REVIEW

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnection.java [hunk 1]
```diff
@@ -575,6 +580,22 @@ public boolean sslModeEnabled() {
             return sslMode() != SecureConnectionMode.DISABLED;
         }
 
+        public String sslKeyStore() {
+            return config.getString(MySqlConnectorConfig.SSL_KEYSTORE);
+        }
+
+        public String sslKeyStorePassword() {
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnection.java:587]: Let's return it as `char[]` right here (same for trust store PW).

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnection.java [hunk 2]
```diff
@@ -502,7 +492,22 @@ public MySqlConnectionConfiguration(Configuration config) {
             final Builder jdbcConfigBuilder = dbConfig
                     .edit()
                     .with("connectTimeout", Long.toString(getConnectionTimeout().toMillis()))
-                    .with("useSSL", Boolean.toString(useSSL));
+                    .with("sslMode", sslMode().name());
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnection.java:495]:
```suggestion
                    .with("sslMode", sslMode().getValue());
```

I.e. let's not rely on the enum constant name, but the assigned value.

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/legacy/MySqlJdbcContext.java [hunk 3]
```diff
@@ -73,7 +73,22 @@ public MySqlJdbcContext(MySqlConnectorConfig config) {
         Builder jdbcConfigBuilder = jdbcConfig
                 .edit()
                 .with("connectTimeout", Long.toString(config.getConnectionTimeout().toMillis()))
-                .with("useSSL", Boolean.toString(useSSL));
+                .with("sslMode", sslMode().name());
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/legacy/MySqlJdbcContext.java:76]: Same as above.

**Review by gunnarmorling — COMMENTED** [review]
LGTM overall, some minor comments inline. And I'll log a follow-up issue to get rid of the duplications between `MySqlJdbcContext` and `MySqlConnection`.

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/legacy/MySqlJdbcContext.java (hunk 3 above)
Comment (harveyyue) [reply debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/legacy/MySqlJdbcContext.java]: fix these issue.

**Comment by gunnarmorling** [comment]
LGTM, applied. Thanks a lot, @harveyyue!

**Pull request merged**